### PR TITLE
Chore: update typography

### PR DESCRIPTION
**Pull request by aliakseilatyp**
Thanks for submitting a PR! Please check the boxes below:

- [ ] I have run [`pre-commit`](https://docs.flagsmith.com/deployment/locally-api#pre-commit) to check linting
- [ ] I have filled in the "Changes" section below?
- [ ] I have filled in the "How did you test this code" section below?

## Changes

- updated typography (size, family, weight and line heights)
- added sass variables for text

## How did you test this code?

_Please describe._

### CODE REVIEW

File: frontend/web/styles/project/_alert.scss [hunk 1]
```diff
@@ -25,11 +25,10 @@
 .alert {
     border-radius: $default-border-radius;
     .title {
-        font-size: 16px;
-        font-weight: bold;
+        font-size: 14px;
         margin-bottom: 5px;
         .ion {
-            font-size: 18px;
+            font-size: 16px;
```
Comment (kyle-ssg) [inline frontend/web/styles/project/_alert.scss]: Is this $font-size-base ?

File: frontend/web/styles/project/_alert.scss [hunk 2]
```diff
@@ -25,11 +25,10 @@
 .alert {
     border-radius: $default-border-radius;
     .title {
-        font-size: 16px;
-        font-weight: bold;
+        font-size: 14px;
```
Comment (kyle-ssg) [inline frontend/web/styles/project/_alert.scss]: $font-size-sm

File: frontend/web/styles/project/_buttons.scss [hunk 3]
```diff
@@ -294,7 +294,6 @@ button.btn {
   box-shadow: none;
   padding: 0;
   line-height: 22px;
```
Comment (kyle-ssg) [inline frontend/web/styles/project/_buttons.scss]: variable

File: frontend/web/styles/components/_aside.scss [hunk 4]
```diff
@@ -413,6 +413,7 @@ $animation-timing-function: ease-in-out;
     &__project-title {
         color: $aside-nav-color;
         font-size: 24px;
+        line-height: 32px;
```
Comment (kyle-ssg) [inline frontend/web/styles/components/_aside.scss]: variables

File: frontend/web/styles/project/_alert.scss (hunk 1 above)
Comment (aliakseilatyp) [reply frontend/web/styles/project/_alert.scss]: done

File: frontend/web/styles/project/_alert.scss (hunk 2 above)
Comment (aliakseilatyp) [reply frontend/web/styles/project/_alert.scss]: done

File: frontend/web/styles/project/_buttons.scss (hunk 3 above)
Comment (aliakseilatyp) [reply frontend/web/styles/project/_buttons.scss]: done

File: frontend/web/styles/components/_aside.scss (hunk 4 above)
Comment (aliakseilatyp) [reply frontend/web/styles/components/_aside.scss]: done

**Review by kyle-ssg — APPROVED** [review]

**Pull request merged**